cxx-qt-gen/src/writer/cpp/mod.rs

```diff
@@ -164,6 +171,7 @@ mod tests {
           ~MyObject();
           const MyObjectRust& unsafeRust() const;
           MyObjectRust& unsafeRustMut();
+          std::unique_ptr<MyObjectCxxQtThread> qtThread();
```

Comment:
if we can make this a `const` method then we don't need to use the `as_mut()` on the Rust side and can just do `self.qt_thread()` ... as this isn't really mutating anything in the `qtThread` call, just incrementing the shread_ptr reference counters. So might make sense to make them `mutable` and allow this to be `const` ? (i'll try it ...)

Reply:
Seems to work without making `mutable` :-)